packages/communication-react/src/mergedHooks.ts

```diff
@@ -134,5 +140,9 @@ export const usePropsFor = <Component extends (props: any) => JSX.Element>(
     return { ...callProps, ...callingHandlers } as any;
   }
 
-  throw "Can't find corresponding selector for this component. Please check the supported components from Azure Communication UI Feature Component List.";
+  if (!chatSelector && !callingSelector) {
+    throw "Can't find corresponding selector for this component. Please check the supported components from Azure Communication UI Feature Component List.";
+  } else {
+    throw 'Get undefined props from useSelectors, please make sure the component is wrapped by providers.';
```

Comment:
`Get undefined props from useSelectors` doesn't quite read correct, 
```suggestion
Could not find props for this component, ensure the component is wrapped by appropriate providers.
```